zk_toolbox/crates/zk_inception/src/commands/prover/generate_sk.rs

```diff
@@ -0,0 +1,27 @@
+use anyhow::Ok;
+use common::{cmd::Cmd, logger, spinner::Spinner};
+use config::EcosystemConfig;
+use xshell::{cmd, Shell};
+
+use super::utils::get_link_to_prover;
+use crate::messages::{MSG_GENERATING_SK_SPINNER, MSG_SK_GENERATED};
+
+pub(crate) async fn run(shell: &Shell) -> anyhow::Result<()> {
+    let ecosystem_config = EcosystemConfig::from_file(shell)?;
+    let link_to_prover = get_link_to_prover(&ecosystem_config);
+    shell.change_dir(&link_to_prover);
+
+    let spinner = Spinner::new(MSG_GENERATING_SK_SPINNER);
+    let mut cmd = Cmd::new(cmd!(
+        shell,
+        "cargo run --features gpu --release --bin key_generator -- 
+            generate-sk all --recompute-if-missing 
+            --setup-path=vk_setup_data_generator_server_fri/data 
+            --path={link_to_prover}/vk_setup_data_generator_server_fri/data"
```

Comment:
should it be absolutize?

Reply:
i'm just worrying, that we are already in `prover` subdir and we want to go to prover subdir one more time

Reply:
`link_to_code` is already an absolute path and `link_to_prover` appends `/prover`
Shoud we still use `absolutize()`?

https://github.com/matter-labs/zksync-era/blob/main/zk_toolbox/crates/config/src/ecosystem.rs#L70